src/components/NavigationBarAside.tsx

```diff
@@ -0,0 +1,40 @@
+import Image from 'next/image';
+import Link from 'next/link';
+import { useState } from 'react';
+import NavigationBarAsideItems from './NavigationBarAsideItems';
+
+const NavigationBarAside = () => {
+  const [isLoggedIn, setIsLoggedIn] = useState(true);
+
+  return (
+    <aside className="navigation-bar-aside">
+      <div className="navigation-bar-aside__wrapper">
```

Comment:
Question, what is the purpose of this wrapper? I don't see why the CSS styling can't be in the `navigation-bar-aside` class itself 🤔